### PR TITLE
chore: set max-line-length in stylelint so auto-formatters pick it up

### PR DESCRIPTION
**Pull request by mmalerba**

### CODE REVIEW

File: test/screenshot/.stylelintrc.yaml [hunk 1]
```diff
@@ -3,6 +3,7 @@ rules:
   # CSS
   custom-media-pattern: "^(?:test|custom|report)-.+"
   custom-property-pattern: "^(?:test|custom|report)-.+"
+  max-line-length: 120
```
Comment (kfranqueiro) [inline test/screenshot/.stylelintrc.yaml]: I think you probably meant to add this to the root-level `.stylelintrc.yaml`, not just the one under `test/screenshot`?

Comment (mmalerba) [reply test/screenshot/.stylelintrc.yaml]: whoops, yes

**Comment by mmalerba** [comment]
Oh this apparently causes some lint failures due to existing lines that are too long, I'll fix those and add them to this PR

**Comment by kfranqueiro** [comment]
Fixed a few nits and updated from master. Running screenshot tests locally right now. Diffed CSS and only diffs were whitespace introduced inside the wrapped `calc` expressions in toolbar.

**Pull request merged**